tests/e2e/e2e_test.go

```diff
@@ -177,28 +164,40 @@ func testAttachWriteReadDetach(volumeID, volName, nodeID string, readOnly bool)
 
 }
 
-func waitForVolumes(params *ec2.DescribeVolumesInput, nVolumes int) {
+func waitForVolumeState(volumeID, state string) {
 	backoff := wait.Backoff{
-		Duration: 60 * time.Second,
-		Factor:   1.2,
-		Steps:    21,
+		Duration: 1 * time.Second,
+		Factor:   1.8,
```

Comment:
This value looks a bit tricky. Why is 1.8 better than 1.2? Could you add some docs around this?